web-api/src/Kerbero.Identity/Modules/Authentication/Services/AuthenticationService.cs

```diff
@@ -21,51 +25,61 @@ IAuthenticationHelper authenticationHelper
     _authenticationHelper = authenticationHelper;
   }
 
-  public async Task<AuthenticatedDto> LoginWithEmail(LoginEmailDto loginEmailDto)
+  public async Task<AuthenticatedDto> LoginWithEmail(LoginDto loginDto)
```

Comment:
```suggestion
  public async Task<AuthenticatedDto> LoginWithEmail(LoginDto loginDto)
```
why the methos si not renamed? if you want to remove the "Emai" part do it consistently